tsdfileapi/resumables.py

```diff
@@ -682,7 +682,8 @@ def merge_chunk(
                     shutil.copyfileobj(fin, fout)
             chunk_size = os.stat(chunk).st_size
             assert self._db_update_with_chunk_info(upload_id, chunk_num, chunk_size)
-        except Exception as e:
+        except (sqlite3.OperationalError, OperationalError, Exception) as e:
```

Comment:
I don't think you need to include sqlite3-specific error classes in the `except` here because [they all inherit from `Exception`](http://docs.python.org/3/library/sqlite3.html#sqlite3.Error) anyway, which inherits from the built-in `Exception`.